### PR TITLE
Sahara: Add Initial barclamp

### PR DESCRIPTION
**Pull request by abonillasuse**
The barclamp was a fork from the magnum initial barclamp request #360

### CODE REVIEW

File: bin/crowbar_sahara [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
```
Comment (tboerger) [inline bin/crowbar_sahara]: It doesn't make sense to add this file anymore, all those files will get dropped for next crowbar release in favor of crowbarctl.

File: crowbar_framework/app/views/barclamp/sahara/_edit_attributes.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= attributes_for @proposal do
+  .panel-sub
+    = header @dep_raw, true
+
+  .panel-body
+    = select_field :foo, :collection => [['XKCD','xkcd'], ['Dilbert', 'dilbert']]
```
Comment (tboerger) [inline crowbar_framework/app/views/barclamp/sahara/_edit_attributes.html.haml]: This needs to be replaced with real fields or just a note that we currently don't provide any flag.

**Comment by tboerger** [comment]
Please fix the naming of the classes, otherwise it's just broken and not usable.

**Comment by dirkmueller** [comment]
superseded by #480

**Pull request closed without merging**